### ISSUE TITLE
A problem with the model

### ISSUE THREAD

**Issue by YUESAI** [ma-compbio/SPEID#1]
Hello.
I ran your code and found that  if use Adam(lr=1e-4) in training the model can fit better , such as IMR90 (precision =0.8594 ,recall =0.9992 , f1 =0.9240 , acc =0.9921 ). While other parameters(num_epoch, training_frac, ...) are same.
Why do you not choose a better result in your paper?
Thank you for your time and I look forward to your reply.

**Comment by sss1** [comment]
Hi @YUESAI ,
We tuned the model hyper-parameters (learning rate, deep network architecture/size, etc.), on the K562 cell line, so the hyper-parameters might be sub-optimal on other cell lines.
